File: openapi/code_samples/python/giftcards/get_product_by_iso/code.py
import requests

url = "https://giftcards.reloadly.com/countries/ES/products"

payload={}
headers = {
  'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
  'Accept': 'application/com.reloadly.giftcards-v1+json'
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)

File: openapi/code_samples/nodejs/giftcards/get_product_by_id/code.js
var request = require('request');
var options = {
  'method': 'GET',
  'url': 'https://giftcards.reloadly.com/products/10',
  'headers': {
    'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
    'Accept': 'application/com.reloadly.giftcards-v1+json'
  }
};
request(options, function (error, response) {
  if (error) throw new Error(error);
  console.log(response.body);
});